src/interfaces/meta-info.ts

```diff
@@ -11,21 +11,46 @@ export interface MetaInfo {
     documentationLink: string;
 }
 
-interface L1Props2 {
-    name?: string;
-}
+export type ContentTypeUID =
+    | 'l1_products'
+    | 'l2_products'
+    | 'programming_languages'
+    | 'technologies'
+    | 'levels'
+    | 'author_types'
+    | 'spoken_languages'
+    | 'content_types'
+    | 'code_levels'
+    | 'event_types'
+    | 'event_attendances';
+
+export const contentTypeUIDtoTagType = new Map<ContentTypeUID, TagType>([
+    ['l1_products', 'L1Product'],
+    ['l2_products', 'L2Product'],
+    ['programming_languages', 'ProgrammingLanguage'],
+    ['technologies', 'Technology'],
+    ['levels', 'ExpertiseLevel'],
+    ['author_types', 'AuthorType'],
+    ['spoken_languages', 'SpokenLanguage'],
+    ['content_types', 'ContentType'],
+    ['code_levels', 'CodeLevel'],
+    ['event_types', 'EventType'],
+    ['event_attendances', 'EventAttendance'],
+]);
 
-interface L1Props {
-    l_1_product?: L1Props2;
+interface L1ProductsConnection {
+    edges: { node: { title: string } }[];
 }
```

Comment:
This is just how CS represents relations, so we have to conform to this structure.